frontend/src/metabase/setup/actions.ts

```diff
@@ -175,7 +176,15 @@ export const skipDatabase = createAsyncThunk(
 export const SUBMIT_USER_INVITE = "metabase/setup/SUBMIT_USER_INVITE";
 export const submitUserInvite = createAsyncThunk(
   SUBMIT_USER_INVITE,
-  (_: InviteInfo, { dispatch }) => {
+  async (inviteInfo: InviteInfo, { dispatch }) => {
+    await dispatch(
+      userApi.endpoints.createUser.initiate({
+        email: inviteInfo.email,
+        first_name: inviteInfo.first_name || undefined,
+        last_name: inviteInfo.last_name || undefined,
+        source: "setup",
+      }),
+    );
```

Comment:
you need an `.unwrap()` here if you want this to throw an error if this request fails